src/plugins/charts/public/static/components/legend_toggle.tsx

```diff
@@ -65,7 +64,6 @@ const LegendToggleComponent = ({ onClick, showLegend, legendPosition }: LegendTo
         defaultMessage: 'Toggle legend',
       })}
       aria-expanded={showLegend}
-      aria-controls={legendId}
```

Comment:
ℹ️ This aria property is used with the wrong way (the dashboard a11y tests were also complaining)